django_project/api/api_views/v2/facilities/detail.py

```diff
@@ -52,3 +55,26 @@ def get(self, request, osm_type, osm_id):
                 raise Http404()
         else:
             return HttpResponseBadRequest('%s is not recognized as osm type' % osm_type)
+
+    def put(self, request):
```

Comment:
hi @myarjunar 
could we still keep `osm_type, osm_id` in parameters? so we can make sure it is right one

and also this parameters into data